src/main/kotlin/no/kommune/oslo/maskinporten/cli/CreateClientCommand.kt

```diff
@@ -13,8 +13,13 @@ class CreateClientCommand : AdminCommand(name = "client") {
     override fun run() {
         super.run()
 
-        val clientId = getAdminClient().createClient(name, description, scopes)
-        println("Client id: $clientId")
+        try {
+            val clientId = getAdminClient().createClient(name, description, scopes)
+            log.info("Created Maskinporten client with id $clientId")
+            println("""{"client_id": "$clientId"}""")
```

Comment:
redundant print statement?

Reply:
The `log` output goes to `stderr` while the `println` goes to `stdout` and can be piped to other commands, e.g. `jq`.